packages/modular-scripts/src/program.ts

```diff
@@ -84,22 +84,64 @@ program
   )
   .option('--verbose', 'Run yarn commands with --verbose set')
   .option('--private', 'Enable the building of private packages', false)
+  .option(
+    '--changed',
+    'Build only for workspaces that have changed compared to the branch specified in --compareBranch',
+    false,
+  )
+  .option(
+    '--descendants',
+    'Additionally build workspaces that descend from the specified packages (can be combined with --changed)',
```

Comment:
Given that the command is called descendants i wouldn't use descend in the description - would be more useful to say 'workspaces which depend on the specified packages' 
Same to --ancestors:
'workspaces that the specified packages depend on'

Reply:
Done